src/app/api/v1/applications/[appSlug]/tickets/deploy/route.tsx

```diff
@@ -77,7 +77,7 @@ async function postHandler(req: NextRequestWithDeveloperUserAccessToken & NextRe
         name: contractName,
         developerId: req.developerId,
         version: nextId,
-        metadataUrl,
+        metadataImgUrl: metadataUrl,
```

Comment:
nope, metadataUrl should stay the same.

Look at the return value from :
`const { metadataUrl, metadataImageUrl } = await uploadMetadata(`


iin the smartContractModel.create, you should go:

```
metadataUrl,
metadataImgUrl: metadataImageUrl
```

you can changed the name of the returned ImgUrl from `uploadMetadata` if you want it to match.